dvc/command/experiments.py

```diff
@@ -789,6 +790,62 @@ def run(self):
         return 0
 
 
+class CmdExperimentsInit(CmdBase):
+    CODE = "src"
+    DATA = "data"
+    MODELS = "models"
+    DEFAULT_METRICS = "metrics.json"
+    DEFAULT_PARAMS = "params.yaml"
+    PLOTS = "plots"
+    DVCLIVE = "dvclive"
+    DEFAULT_NAME = "default"
+
+    def run(self):
+        from dvc.command.stage import parse_cmd
+
+        cmd = parse_cmd(self.args.cmd)
+        if not cmd:
+            raise InvalidArgumentError("command is not specified")
+        if self.args.interactive:
+            raise NotImplementedError(
+                "'-i/--interactive' is not implemented yet."
+            )
+        if self.args.explicit:
+            raise NotImplementedError("'--explicit' is not implemented yet.")
+        if self.args.template:
+            raise NotImplementedError("template is not supported yet.")
+
+        from dvc.utils.serialize import LOADERS
+
+        code = self.args.code or self.CODE
+        data = self.args.data or self.DATA
+        models = self.args.models or self.MODELS
+        metrics = self.args.metrics or self.DEFAULT_METRICS
+        params_path = self.args.params or self.DEFAULT_PARAMS
+        plots = self.args.plots or self.PLOTS
+        dvclive = self.args.live or self.DVCLIVE
+
+        _, ext = os.path.splitext(params_path)
+        params = list(LOADERS[ext](params_path))
+
+        name = self.args.name or self.DEFAULT_NAME
+        stage = self.repo.stage.add(
+            name=name,
+            cmd=cmd,
+            deps=[code, data],
+            outs=[models],
+            params=[{params_path: params}],
+            metrics_no_cache=[metrics],
+            plots_no_cache=[plots],
+            live=dvclive,
```

Comment:
Default stage can ignore `--dvclive` arg. We probably need conditional arg parsing based on the template, throwing an error if a provided argument is irrelevant for that template.

Reply:
We don't have a default stage/template yet, so I left it with the default. I'll change this when we have templates.